modules/pages-and-forms/pages/customize-display-process-monitoring.adoc

```diff
@@ -0,0 +1,54 @@
+= How to customize the display of process execution monitoring
+:page-aliases: ROOT:customize-display-process-monitoring.adoc
+:description: Learn how to modify the process/case visualization page using the UI Designer, or embed the BPMN diagram fragment in your own page.
```

Comment:
Something is missing to indicate that this feature is only available in efficiency, performance and enterprise subscription editions